src/domain/Contact.js

```diff
@@ -291,7 +291,11 @@ export default class Contact {
   }
 
   isInCall(): boolean {
-    return this.status === 1;
+    // Codes AST
+    const INUSE = 1; // 1 << 0, One or more devices INUSE
```

Comment:
Can you move these constant outside of the function ?
It's not necessary re-declare them each time

Reply:
Done